lib/common/common/Jobs.inl

```diff
@@ -52,7 +52,7 @@ namespace JsUtil
     template<class TJobManager, class TJobHolder>
     void JobProcessor::PrioritizeJobAndWait(TJobManager *const manager, const TJobHolder holder, void* function)
     {
-        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>;
+        TemplateParameter::SameOrDerivedFrom<TJobManager, WaitableJobManager>();
```

Comment:
Don't both of these versions run the default constructor of the object? Or is there a subtle difference here or a complaint from clang?